### PR TITLE
5.3 works now, so no need to skip 5.3, just upstream

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

File: cfme/tests/automate/test_namespace.py [hunk 1]
```diff
@@ -20,11 +21,16 @@
...
 def namespace(request):
     # don't test with existing paths on upstream (there aren't any)
-    if request.param is ta.a_namespace_with_path and version.current_version() >= "5.3":
+    if request.param is ta.a_namespace_with_path and version.current_version() == version.LATEST:
         pytest.skip("don't test with existing paths on upstream (there aren't any)")
     return request.param()
 
 
+@pytest.fixture
+def setup_single_provider():
```
Comment (seandst) [inline cfme/tests/automate/test_namespace.py]: Seems like this should be in `fixtures.providers`, I keep seeing examples of it in PRs.

Comment (psav) [inline cfme/tests/automate/test_namespace.py]: Yeh it really should, can I do that in a new PR?

**Comment by mfalesni** [comment]
Lint report for commit 4d9fe52064e0ba0bd7f2b5da35620dc6751a243f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/automate/test_namespace.py (hunk 1 above)
Comment (seandst) [inline cfme/tests/automate/test_namespace.py]: Yeah, certainly

**Pull request merged**